src/common/logger.ts

```diff
@@ -135,6 +141,29 @@ export class Logger {
         }
     }
 
+    /**
+     * This will send the logs to the client if loggerWindow
+     * else adds the logs to a Queue
+     *
+     * @param logMsg {ILogMsg}
+     */
+    public sendToCloud(logMsg: ILogMsg): void {
+        // don't send logs if it is not desired by the user
+        if (this.desiredLogLevel && this.desiredLogLevel !== logMsg.level) {
+            return;
+        }
+
+        if (this.loggerWindow) {
```

Comment:
let's use `return` here as well to keep it consistent? Please remove the `else` statement.